pemFioi/quickAlgo/interface-mobileFirst.js

```diff
@@ -266,6 +269,53 @@ var quickAlgoInterface = {
         }
     },
 
+    loadXmlSubject: function(xml) {
+        var subjectNodes = xml.getElementsByTagName("subject");
+        if (subjectNodes.leading != 0) {
+            var subject = subjectNodes[0];
+            document.title = subject.getAttribute("title");
+            $(".exerciseText").text(subject.getAttribute("description"));
+        }
+    },
+
+    openEditExercise: function() {
+
+        var description = $(".exerciseText").text();
+
+        var editExerciseHtml = `
```

Comment:
Template literals aren't supported in older browsers.

Reply:
I found these templates in QuickPi also